index.bs

```diff
@@ -2916,6 +2916,214 @@ value and terminate the operation.
 
 </div>
 
+### Signal Credential Changes to the Authenticator - PublicKeyCredential's `signal()` Method {#sctn-signal}
+
+<xmp class="idl">
+    partial interface PublicKeyCredential {
+        static Promise<undefined> signal(PublicKeyCredentialSignalOptions options);
+    };
+
+    dictionary PublicKeyCredentialSignalOptions {
+        USVString                              rpId;
+        Base64URLString                        unknownCredentialId;
+        PublicKeyCurrentCredentialDetails      currentCredentials;
+    };
```

Comment:
What if we namespaced reports here, so that every type of report is encapsulated in some way? Sure, a report might only have one value. But I imagine it'd go more smoothly for future-us, when we inevitably want to tweak values to existing reports, if we start defining all reports now as dictionaries.

e.g. I might suggest rearranging these two reports like this:

```
const signalOptions = {
  unknownCredential: { rpId: "...", id: "..." },
  currentCredentials: { rpId: "...", userId: "...", ... },
};
```

It's a little bit verbose, but I think it'd be easier for RPs to drill down to the specific report they want and define only that report's values within a dictionary.

Reply:
I like this! Done.